public/css/customv600.css

```diff
@@ -329,4 +343,57 @@
 }
 .floatRight{
     float:right
-}
\ No newline at end of file
+}
+
+/* Dark theme styles */
```

Comment:
Thank you so much for this. Here are the list of bugs I have found with dark mode so far. Let me know which ones you are able to work on. I can pick up the rest.

**High priority**
- [ ] main logo is black in color. I already have white logo added in assets https://github.com/openingtree/openingtree/blob/master/public/opening-tree-logo-white.png. can we use this one?
- [ ] "Black" and "White" text in "colors and filters" accordion is not visible. 
- [ ] "Advanced filters" link is misplaced
- [ ] "view moves>>" action below "White games loaded: 53" text is very hard to read
- [ ] Footer of moves table "Showing moves that have been played by vannooz in this position. vannooz is playing as white."is still black in color
- [ ] The table in "report" tab has black text


**Medium Priority**
- [ ] "Change" "stop" and other blue links are hard to see
- [ ] "Analyze games" Primary call to action does not stand out compared to other actions.
- [ ] "White games loaded: 53" text is really hard to read
- [ ] "Book indicators", enabled using the "wrench" icon are not visible


**Low Priority**
- [ ] Overlays like FAQ and acknowledgements under "help" and "more" sections are still white
- [ ] Success/Failure error messages in snackbar are all the same color
- [ ] Notable player name, once selected, is very hard to read